src/sphinx/general/scenario.rst

```diff
@@ -325,6 +325,8 @@ Similar to ``doSwitch``, but with a fallback if no switch is selected.
 ``randomSwitch`` can be used to emulate simple Markov chains.
 Simple means cyclic graphs are not currently supported.
 
+.. warning:: Beware: you have to use parentheses when using these switch methods, curly braces will change the semantic and will not work as expected.
```

Comment:
What do you mean by "change the semantic" and "no work as expected"? It's not like it would seem to work but actually do something different than expected. It simply won't compile.